pkg/operator/ceph/object/objectstore.go

```diff
@@ -195,6 +195,12 @@ func deleteSingleSiteRealmAndPools(objContext *Context, spec cephv1.ObjectStoreS
 // This is used for quickly getting the name of the realm, zone group, and zone for an object-store to pass into a Context
 func getMultisiteForObjectStore(clusterdContext *clusterd.Context, spec *cephv1.ObjectStoreSpec, namespace, name string) (string, string, string, error) {
 	ctx := context.TODO()
+
+	// TODO: check or fix https://github.com/rook/rook/issues/6342
```

Comment:
Why do we need to mention this issue?

Reply:
If we are fixing #6342, this condition may be moved below right?